src/app/data-evaluation/facility/analysis/run-analysis/group-analysis/group-analysis-options/group-analysis-options.component.html

```diff
@@ -4,6 +4,14 @@
             (click)="goToMeterGroups()">Meter Grouping Page</a> to add
         meters to this group for analysis.
     </div>
+
+    <div class="form-check mt-5">
+        <input class="form-check-input checkbox-input" type="checkbox" id="excludeGroup" name="excludeGroup"
+            [(ngModel)]="group.isGroupExcluded" (ngModelChange)="setExcludeGroup()">
```

Comment:
I like the approach here but instead of adding a new variable lets just use what we already have with the analysisType and value "skip"